tools/build.xml

```diff
@@ -82,6 +56,8 @@
     <target name="build" depends="copper-generate">
         <mkdir dir="${build.dir}"/>
         <javac srcdir="${src.dir}" destdir="${build.dir}" includeantruntime="false">
+            <compilerarg value="-Xlint:deprecation"/>
```

Comment:
This dude is the cause of a lot of new deprecation warnings. That's because a lot of the material is deprecated but still compiled.
The intention is to recursively deprecate, then delete this line and the rest of the deprecated code.